sites/funcorporate/src/backend/funcorporate/settings.py

```diff
@@ -327,13 +327,13 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         (
             "richie.apps.search.filter_definitions.IndexableMPTTFilterDefinition",
             {
-                "human_name": _("Subjects"),
+                "human_name": _("Training domains"),
                 "is_autocompletable": True,
                 "is_searchable": True,
                 "min_doc_count": 0,
-                "name": "subjects",
+                "name": "domains",
```

Comment:
Is it normal that with demo site data, this new filter is empty?

Reply:
Yes. It requires a category page with a reverse_id of "domains". I submitted a PR on richie to improve demo site on customized sites and will look into it once we have released the next version of richie.